modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase

```diff
@@ -683,6 +683,71 @@ definition {
 			assigneeType = "Organizations");
 	}
 
+	@description = "This test asserts that an Account Admin can unassign roles to users."
+	@priority = "5"
+	test AccountAdminCanUnassignRolesToUsers {
+		task ("Given an Account User is given a role by the Account Administrator") {
+			JSONAccountEntryUser.addAccountEntryUser(
+				accountEntryName = "Account Name",
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "user1",
+				userLastName = "user1",
+				userScreenName = "user1");
+
+			LexiconEntry.gotoEntry(rowEntry = "Account Name");
+
+			Navigator.gotoNavTab(navTab = "Users");
+
+			Accounts.goToEntryMenuItem(
+				menuItem = "Assign Roles",
+				rowEntry = "user1 user1");
+
+			SelectFrame(
+				key_title = "Assign Roles",
+				locator1 = "IFrame#MODAL_ANY");
+
+			Check(
+				key_tableEntry = "Buyer",
+				locator1 = "LexiconTable#TABLE_ENTRY_CHECKBOX");
+
+			SelectFrameTop();
+
+			Button.click(button = "Done");
+
+			AssertTextEquals(
+				key_columnName = "lfr-account-roles-column",
+				key_rowEntry = "user1 user1",
+				locator1 = "ContentRow#ENTRY_CONTENT_ROW_NUMBER_TD",
+				value1 = "Buyer");
+		}
+
+		task ("When the Account Administrator unassigns that role") {
+			Accounts.goToEntryMenuItem(
+				menuItem = "Assign Roles",
+				rowEntry = "user1 user1");
+
+			SelectFrame(
+				key_title = "Assign Roles",
+				locator1 = "IFrame#MODAL_ANY");
+
+			Uncheck(
```

Comment:
See if this can be combined in Assign Roles macro, if not, we can add a new macro for unassign role as well.